portfolio.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Alicja Zubel Developer</title>
+	<link rel="icon" href="favicon.ico" type="image/x-icon"/>
+	<link rel="stylesheet" href="normalize.css" type="text/css">
+	<link rel="stylesheet" href="style.css" type="text/css">
+	<link rel="preconnect" href="https://fonts.gstatic.com">
+	<link href="https://fonts.googleapis.com/css2?family=Abril+Fatface&display=swap" rel="stylesheet">
+	<link rel="preconnect" href="https://fonts.gstatic.com">
+	<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet"> 	
+</head>
+<body>
+	<div class="container">
+		
+		<section class="menu">
+			<div class="logo">
+				<button class="menu-image">&lt;</button>
+				<a href="index.html"> <img class="menu-logo" src="karate.jpg" alt="logo"> </a>
+			</div>
+			<div class="navigation">
+				<nav>
+					<a href="index.html">Home</a>
+					<a href="about.html">About</a>
+					<a class="current" href="portfolio.html">Portfolio</a>
+					<a href="contact.html">Contact</a>
+					<a href="">CV</a>
+				</nav>
+			</div>
+			<div class="links">
+				<a href="https://github.com/AlicjaZub" target="_blank"><img class="menu-image" src="githublight.png" alt="github logo"></a>
+				<a href="https://www.linkedin.com/in/alicja-zubel-07a9515a/" target="_blank"><img class="menu-image" src="lin.png" alt="linkedin logo"></a>
+			</div>
+		</section>
+		
+		<section class="right-section">  <!-- to be adjusted with projects -->
+			<button class="hidden-menu-button">MENU</button>
+			<div class="porfolio-heading">
+				<h1 class="welcome-heading" >Projects</h1>
+			</div>
+			<div class="info info-portfolio" >
+				<div class="left project-left">
```

Comment:
`project-left` is not very semantic, try using `project-img` then no class on the `<img>` tag, and in the css you can use `.project-img img {}`